src/ProjectNeon/Assets/Z_Tests/Features/Battle/StatsDecorationTests.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Linq;
+using NUnit.Framework;
+
+public class StatsDecoRationTests
+{
+    [Test]
+    public void Stats_Barkskin_DecorateArmor()
+    {
+        var memberStats = TestableObjectFactory.Create<MemberState>();
+        var spellStats = TestableObjectFactory.Create<BarkskinStats>();
+        spellStats.Init(memberStats);
+        memberStats.CurrentStats = spellStats;
+        Assert.IsTrue(spellStats.Armor().Equals(10.0F));
```

Comment:
This is a good test. 

Make sure that `BarkskinStats` is used only as type `BattleStats` to prevent the problem with `new`

Reply:
@SilasReinagel How could I do that?